### ISSUE TITLE
 background URL not working correctly

### ISSUE THREAD

**Issue by Angelxoxo757** [microsoft/vscode#180359]
The background URL it's not working at all at first it was working correctly, but all of a sudden it doesn't work. No matter how much I've updated the program still the same result. This is not my first time using Visual Studio Code, if there is a fix, please provide steps how to fix the issue. Also, the image source capability it's not cooperating at all when I add a class within my css code.

**Comment by roblourens** [comment]
I don't know what you are referring to but it sounds like an issue with your code. Please explain more if you think vscode is doing the wrong thing.

**Comment by Angelxoxo757** [comment]
OK, sorry for the late reply. Here is the issue every time I use the css property background URL and I put the location of the file it doesn't show up. I've also tested to see if there is a issue with my code there's not with the other applications I made, no images are you showing up within the previous applications I've made on my system. I find it strange because i
!t worked previously but now it doesn't show up at all. I'll send you a screenshot of my code. I only received is the issue when I use background URL I've looked all around the Internet and I don't know how to fix the issue, so I figured it was a bug within Visual Studio Code. That's why I contacted you all.

![Screenshot 2023-04-20 at 4 31 45 PM](https://user-images.githubusercontent.com/112197120/233481550-98b4f8ee-9438-4d70-9305-f20ced51197f.png)
![Screenshot 2023-04-20 at 4 32 18 PM](https://user-images.githubusercontent.com/112197120/233481663-66843c6a-4398-4ab5-9164-50caaf875f83.png)
